### DISCUSSION TITLE
Unsupported option "plugin_languages_indepth"

### DISCUSSION THREAD

**Discussion by andry81** [lowlighter/metrics#1145] [Q&A] [answered]
The output svg contains the error inside.

URL: `metrics.lecoq.io/andry81?template=classic&languages=1&lines=1&languages.limit=12&languages.threshold=0%25&languages.colors=github&languages.sections=most-used&languages.details=bytes-size%2C%20percentage&languages.indepth=true&languages.analysis.timeout=30&languages.categories=markup%2C%20programming&languages.recent.categories=markup%2C%20programming&languages.recent.load=300&languages.recent.days=14&config.timezone=Europe%2FMoscow`

It has started some time ago:
https://github.com/andry81-cache/andry81--gh-content-cache/commit/4c08909630e1eccba022dc128a3dc7244be36ea2#diff-566213d7bf64064d44a7e8b161368c146b7d1d07e403356652f97f7aa40abbd3

Is that a bug?

**Comment by lowlighter** [comment, marked as answer]
Hi 👋 ! 

Yeah since last version the web instance displays a error message if a setting isn't supported, rather than silently ignoring it

The language indepth option is indeed not supported by the shared web instance (because it's too heavy on computation).
With #1141 I'm planning to make it more clear which option are unsupported/disabled on web instance

Sorry for the inconvenience
